### PR TITLE
Add AUDIT_PHPCS_PARALLEL environment variable.

### PR DESCRIPTION
**Pull request by rheinardkorf**
Allows for specifying how many parallel processes to run with phpcs.
Default: 1

### CODE REVIEW

File: app/config/parameters.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 parameters:
+    # Defaults
+    env(AUDIT_PHPCS_PARALLEL): 1
```
Comment (valendesigns) [inline app/config/parameters.yml]: What is this?

Comment (valendesigns) [reply app/config/parameters.yml]: @rheinardkorf We don't need to set a default like this. If the env var is missing it should complain when bringing up the image.

Comment (rheinardkorf) [reply app/config/parameters.yml]: its is symfony's way to include a default.... you don't want a default if its not specified?

Comment (valendesigns) [reply app/config/parameters.yml]: We could consider it, but none of the other variables have a default. So why would this one be special?

Comment (rheinardkorf) [reply app/config/parameters.yml]: because I forgot to set it and my audit server blew up ;)

Comment (valendesigns) [reply app/config/parameters.yml]: We'll add the ones that have a default in another PR.

Comment (valendesigns) [reply app/config/parameters.yml]: Leave this one in there for now to kick things off

**Pull request merged**